server/StudentList.ts

```diff
@@ -0,0 +1,33 @@
+import Student from "./Student.ts"
+
+class StudentList {
+
+    private students: Map<String ,Student>;
+
+    constructor(){
+        this.students = new Map();
+    }
+
+    size(): number{
+        return this.students.size;
+    }
+
+    addStudent(nick: string, name: string, surname: string): void{
```

Comment:
I think you should here pass obj: Student, rather than creating new I this function.
I see two problems here:
- function name is  _addStudent_ not _createAndAddStudent_
- you cant you always create a new _Student_ with a unique id and you cant add an existing _Student_ object

Reply:
alright